exporters/trace/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Definitely should be included, not auto-generated. I needed to add it to be able to mock final methods. See https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods

Reply:
I'd suggest to make the classes / methods non-final instead, it's not a big deal.

Reply:
I'm a bit confused by this. The final method I was talking about above is batchWriteSpans(), not any of my methods/classes.

Reply:
Sorry I missed that this was needed to mock the TraceServiceClient.

The plan was actually to use https://github.com/googleinterns/cloud-operations-api-mock for starting a fake server instead of mocking clients using Mockito or other frameworks.

@james-bebbington WDYT?
It might be fine to merge for now, and replace it with the mock server mentioned above later.

Reply:
Yea good point. I would definitely prefer us to go straight to using the fake server for consistency with the other repos.

Here's an example where this was recently added to the Python exporter: https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/pull/37/files. Before we merge this PR, we should at least get in touch with @the-ericwang35 to see if he has plans to add this to Java or if he can assist you adding in doing so.

Reply:
Yeah, we were actually planning on reaching out to the Java exporter team sometime this week so this works out well. I can definitely help out if needed, it seems like this integration will be similar to the Python one, which also uses `TraceServiceClient`. It might be easier to merge this PR as is, and then I can create a separate PR switching in our mocks, but I'm fine with anything.